### PR TITLE
fix(tests): update snuba route on test

### PR DESCRIPTION
**Pull request by JoshFerge**
fixes broken acceptance test after merge of https://github.com/getsentry/snuba/pull/3157

### CODE REVIEW

File: src/sentry/testutils/cases.py [hunk 1]
```diff
@@ -1376,13 +1376,18 @@ def setUp(self):
         self.addCleanup(patcher.stop)
 
     def drop_replays(self):
-        assert requests.post(settings.SENTRY_SNUBA + "/tests/replays/drop").status_code == 200
+        assert (
+            requests.post(settings.SENTRY_SNUBA + "/tests/entities/replays/insert").status_code
+            == 200
+        )
```
Comment (volokluev) [inline src/sentry/testutils/cases.py:1382]: This is no longer dropping the table, will that work?

Comment (JoshFerge) [reply src/sentry/testutils/cases.py:1382]: ah 🤦🏼

Comment (volokluev) [reply src/sentry/testutils/cases.py:1382]: you shouldn't have to change this actually, the `drop` endpoint is still there in master

Comment (JoshFerge) [reply src/sentry/testutils/cases.py]: yup 👍🏼

**Review by volokluev — APPROVED** [review]

**Review by enochtangg — APPROVED** [review]

**Pull request merged**